e2e/tooltip.spec.ts

```diff
@@ -21,7 +21,7 @@ test("should have correct tooltip content", async ({
   await selectors.edition.drawerLoaded();
 
   const filterLocator = await within(
-    selectors.edition.controlSectionBySubtitle("Filters")
+    selectors.edition.controlSectionByTitle("Filters")
```

Comment:
Question: Do you find this helpers useful ? I recently migrated to not try to have DRY code in my E2E tests in favor of generating them directly with the record feature from Playwright. Actually I think here it makes sense not to have to re-record all those tests.

Reply:
Yes, I find them useful! Here I think the header level changed to always be the same so we didn't need to have this distinction anymore – but I'd also avoid re-doing the tests, especially that it still sometimes happen that the cube stops working and we actually need to make a change 🥹